lib/node_modules/@stdlib/stats/base/dists/logistic/ctor/benchmark/benchmark.js

```diff
@@ -526,11 +626,15 @@ bench( pkg+':quantile', function benchmark( bm ) {
 	mu = 2.0;
 	s = 3.0;
 	dist = new Logistic( mu, s );
+	len = 100;
+	x = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( -3.0, 3.0 );
```

Comment:
```suggestion
		x[ i ] = uniform( 0.0, 1.0 );
```

Same comment as above. This step ensures that the function is not called on values outside its domain.